hoomd/mpcd/stream.py

```diff
@@ -693,6 +693,52 @@ def set_params(self, R=None, boundary=None):
             self.boundary = boundary
 
         bc = self._process_boundary(self.boundary)
-        self._cpp.geometry = _mpcd.SphereGeometry(self.R,bc)
+        self._cpp.geometry = _mpcd.SphereGeometry(self.R,0.0,bc)
         if self._filler is not None:
             self._filler.setGeometry(self._cpp.geometry)
+
+class drying_droplet(_streaming_method):
+    r""" Drying Droplet streaming geometry.
+
+    Args:
+        R (variant): confinement radius
+        density (float): solvent number density inside sphere
+        boundary (str): boundary condition at wall ("slip" or "no_slip")
+        period (int): Number of integration steps between collisions
+
+    The drying_droplet class models a fluid confined inside a drying droplet, centered at the
+    origin and droplet size is decreasing according to radius variant R. Solvent particles are
+    reflected from the spherical walls using appropriate boundary conditions.
+
+    Examples::
+        stream.drying_droplet(period=10, R= hoomd.variant(variant for R), density =5.,seed=394 )
```

Comment:
Examples should actually work!
```suggestion
        R = hoomd.variant.linear_interp([(0, 50), (1e6, 25)])
        stream.drying_droplet(period=10, R=R, density=5., seed=394)
```